components/Project.js

```diff
@@ -31,11 +31,11 @@ export default function Project({ project }) {
 
   return (
     <StyledListItem>
-      <StyledLink href={`/projects/${id}`}>
+      <Link href={`/projects/${id}`}>
         <StyledArticle>
           <StyledTitle>{title}</StyledTitle>
         </StyledArticle>
-      </StyledLink>
+      </Link>
```

Comment:
Now in the mobile worlds things may be blurry, but don't make whole articles clickable (try to copy the title on desktop!).

- make just the text portion a link _inside_ the title
- in the mobile case, you can flex things to get all the possible space, but at least you are desktop safe ^^